operations/tempo-mixin-compiled/dashboards/tempo-reads.json

```diff
@@ -1498,7 +1498,7 @@
  "schemaVersion": 14,
  "style": "dark",
  "tags": [
-  "tempo"
+  "tempo-mixin"
```

Comment:
I would prefer to keep this "tempo" as it shows up nicely in the Grafana Dashboard explore. 
![image](https://user-images.githubusercontent.com/10982987/171411470-d18322ad-c500-423a-8c8a-d5ea382b80db.png)

but no strong opinion!

Reply:
Once nice thing about changing it is that we'd be able to tell that those dashboards came from the mixin.  I was thinking about this in the context when other dashboards get created that aren't part of the mixin, knowing where to make changes requires some head scratching.